tuple-table/src/main/java/com/iluwatar/tuppletable/DBConnection.java

```diff
@@ -0,0 +1,51 @@
+package com.iluwatar.tuppletable;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DBConnection {
+
+    /**
+     * @should @return connection
+     * @throws SQLException
+     * @throws ClassNotFoundException
+     */
+    public Connection getConnection() throws SQLException, ClassNotFoundException, IOException {
+        FileInputStream fis=new FileInputStream("src/main/resources/connection.prop");
```

Comment:
Java 11 enables using `var` keyword for local variables. Please use it where possible.

Reply:
Removed reading from property file, using in memory database as you suggested.